### ISSUE TITLE
Rewrite in Java with JBang?

### ISSUE THREAD

**Issue by bsideup** [bsideup/tinsalver#1]
Although the initial idea was NodeJS heavy, currently the script is rather small and can be rewritten in Java quite easily.

**Comment by aalmiray** [comment]
Indeed. Could even pair it with the jbang GH action to invoke everything on CI ...

**Comment by bsideup** [comment]
Well, the whole point is to not have it on CIs, but... whatever 😂

**Comment by aalmiray** [comment]
Oh I get that. The current cmd requires user/password, is that just for sonatype or also accessing keybase?

**Comment by bsideup** [comment]
Just for Sonatype (to upload the signatures to the staging repo), you need to auth to Keybase CLI separately, and TinSalver delegates the signing to it :)
